src/Microsoft.ML.Data/Transforms/ColumnCopying.cs

```diff
@@ -39,7 +39,7 @@ public sealed class ColumnCopyingEstimator : TrivialEstimator<ColumnCopyingTrans
         {
         }
 
-        public ColumnCopyingEstimator(IHostEnvironment env, params (string source, string name)[] columns)
```

Comment:
>ColumnCopyingEstimator [](start = 15, length = 22)

I'll bug the next person about XML comments, or do it:)

Reply:
Would you trust me if I say what I plan to do that in other PR?
This one is purely IDataTransform hiding.

---
In reply to: [246154879](https://github.com/dotnet/machinelearning/pull/2031#discussion_r246154879) [](ancestors = 246154879)

Reply:
yep!

---
In reply to: [246156509](https://github.com/dotnet/machinelearning/pull/2031#discussion_r246156509) [](ancestors = 246156509,246154879)